### ISSUE TITLE
Sessions tracking issue with Sandbox version of macOS app in 5.2.0

### ISSUE THREAD

**Issue by vdudchak** [getsentry/sentry-cocoa#655]
After updating to 5.2.0, AppStore version of our macOS app stopped to track session.
`
[SentrySDK startWithConfigureOptions:^(SentryOptions *options) {
        options.enableAutoSessionTracking = @YES;
        options.sessionTrackingIntervalMillis = UINT_MAX;
}];
`
Maybe the reason is the UINT_MAX  interval, but it works in non-sandbox versions of the app and it also works in 5.1.9.

**Comment by philipphofmann** [comment]
Hi @vdudchak, thanks for reaching out. `SentryOptions.sessionTrackingIntervalMillis` is an [`NSUInteger`](https://developer.apple.com/documentation/objectivec/nsuinteger?language=objc). `UINT_MAX` comes from C and is defined in `limits.h`. Maybe it helps if you use `NSUIntegerMax` instead.

If you set the `sessionTrackingIntervalMillis` to such a high value a session is only going to be tracked when you close your application or the application crashes. 

Can you please be more specific and tell us exactly how to reproduce this issue?

**Comment by philipphofmann** [comment]
@vdudchak do you have any updates on this issue?

**Comment by vdudchak** [comment]
> @vdudchak do you have any updates on this issue?

Sorry for the late response. No, we are currently using 5.1.9 and everything works fine, I will try to integrate 5.2.2 for the next release and if issue still present will debug to provide more info.

**Comment by philipphofmann** [comment]
Thanks, @vdudchak for the update. Looking forward to hearing from you once you did your next release.

**Comment by philipphofmann** [comment]
@vdudchak any updates on this? FYI we did lots of fixes in 6.0.0 for sessions. It's worth upgrading.

**Comment by philipphofmann** [comment]
As we didn't receive any updates on this one for quite a while, we are going to close this issue. Please feel free to open it again if you have more information or still experience the issue.

**Comment by vdudchak** [comment]
@philipphofmann As it turned out, our problem was not related to the session interval. In our project we used  Carthage to integrate Sentry and we did not set the releaseName in SentryOptions. Our best guess is that `NSDictionary *infoDict = [[NSBundle mainBundle] infoDictionary];` returns nil in release configuration for Sandbox version of the app if Sentry framework is pre-builded with Carthage. 
Maybe it makes sense to mention releaseName in your documentation in case of setup with Carthage. :)
Also the same problem is present in current version of framework (6.2). I was trying to figure out why it worked in 5.1.9, but I hadn't find a reason. Maybe there was no check for releaseName when session was captured back then.
